Sources/Actions/Admin/Permissions.php

```diff
@@ -3258,6 +3258,7 @@ protected static function loadAllPermissions(): void
 
 		foreach (self::getPermissions() as $permission => $perm_info) {
 			// If this permission shouldn't be given to certain groups (e.g. guests), don't.
+			/** @var string $perm */
```

Comment:
Static Analysis hint.

Reply:
😛 Useless clutter.

Reply:
I'm removing it, but you have to consider this will be a major problem for static analysis when we introduce psalm for checking our code.  We know/understand things, but if the static analysis is confused, it is going to have the wrong type of information and warn us about incorrect usages in functions.  The hints help it understand what we know.  This is typically true of arrays where the members of the array are unknown to it unless it can see the array and its members defined earlier.  @Tyrsson can provide a peak, but he said there was quite a bit that psalm will warn about if we introduced it today.

Reply:
Bleh. Fine.